pkg/manila/const.go

```diff
@@ -48,15 +48,17 @@ const (
 	// Manila is the global ServiceType that refers to all the components deployed
 	// by the manila operator
 	Manila storage.PropagationType = "Manila"
-
-	// DefaultsConfigFileName -
+	// DefaultsConfigFileName - Provided by Manila umbrella controller
 	DefaultsConfigFileName = "00-config.conf"
-	// CustomConfigFileName -
-	CustomConfigFileName = "01-config.conf"
-	// CustomServiceConfigFileName -
-	CustomServiceConfigFileName = "02-config.conf"
-	// CustomServiceConfigSecretsFileName -
-	CustomServiceConfigSecretsFileName = "03-config.conf"
+	// ServiceConfigFileName - Represents service config generated in the operator
+	ServiceConfigFileName = "01-config.conf"
+	// CustomConfigFileName - Config snippets inherited by the top-level CR
+	CustomConfigFileName = "02-config.conf"
+	// CustomServiceConfigFileName - Config snippets defined for the sub CR
+	CustomServiceConfigFileName = "03-config.conf"
+	// CustomServiceConfigSecretsFileName - Snippet generated by Secrets passed
+	// to the sub CR
+	CustomServiceConfigSecretsFileName = "04-config.conf"
```

Comment:
this will take some getting used to; but i understand the rationale behind the simple numbered naming here..

Reply:
We can change these numbers to a more self explained and user friendly naming, I'll try on a follow up!